src/image/image.js

```diff
@@ -457,27 +556,37 @@ p5.prototype.encodeAndDownloadGif = function(pImg, filename) {
 };
 
 /**
- * Captures a sequence of frames from the canvas that can be used to create a
- * movie. Frames are downloaded as individual image files by default.
+ * Creates an array of frames from the canvas that can be saved as images.
```

Comment:
How about 'Saves a sequence of frames from the canvas' or 'Captures a sequence of frames from the canvas'?

What do you think, @davepagurek ?

Reply:
"Captures" sounds good to me! (we have "saved" later in the sentence so "captures" avoids duplication)